core/scripts/validate_domain.py

```diff
@@ -61,8 +72,48 @@ def validate_domain() -> int:
             button_mismatches.add(response_id)
 
     if button_mismatches:
-        print(f"Responses with button mismatches: {button_mismatches}")
-        has_errors = True
+        print(f"Button mismatches: {sorted(button_mismatches)}")
+
+    return len(button_mismatches) != 0
+
+
+def validate_translations(english_responses: dict, french_responses: dict) -> bool:
+    missing_translations = set()
+
+    for response_id in english_responses.keys():
+        if response_id not in french_responses:
+            continue
+
+        english_response = english_responses[response_id]
+        french_response = french_responses[response_id]
+
+        english_text = get_text(english_response)
+        french_text = get_text(french_response)
+
+        if len(english_text - french_text) != len(english_text):
```

Comment:
Why not only `if english_text == french_text`? I trust there's a reason, but I don't see it.

Reply:
We are comparing a list of text elements (including the buttons) and we want to output a warning as soon as there is one text element not translated for the response. I will rename the variables (and the method name) to make this more obvious.